packages/translated-components/src/index.js

```diff
@@ -1,27 +1,36 @@
 import React from 'react'
-import {Broadcast, Subscriber} from 'react-broadcast'
+import PropTypes from 'prop-types'
 import reduce from 'lodash/reduce'
 import kebabCase from 'lodash/kebabCase'
 import isString from 'lodash/isString'
 import isNumber from 'lodash/isNumber'
 import IntlFormat from 'intl-messageformat'
 import {countries, currencies} from 'country-data'
 
-const CHANNEL = 'language'
-const DEFAULT_LANG = 'en_US'
-let customDefaultLanguage = DEFAULT_LANG
+export class LocaleProvider extends React.Component {
+  getChildContext () {
+    return {
+      locale: this.props.locale,
+      defaultLocale: this.props.defaultLocale || 'en_AU'
```

Comment:
Could we add one that sets it to something else? We have tests to see that local child props override it, but do we have ones for the React context plumbing? (seeing as they talk about it in such hushed tones).